### PR TITLE
Added Stubs and Versions

### PR DESCRIPTION
**Pull request by ramsainanduri**
- Added stub blocks
- Added versions for all the modules, subworkflows and workflows
- Fixed the indentation issue (4 sapces instead of a tab)
- Config/nextflow.hopper.config re-stuctured
- Corrected the folder structure for nf-core modules from modules/nf-core/modules -> modules/nf-core

### CODE REVIEW

**Comment by ramsainanduri** [comment]
Test run successful with actual sample for hg38_u2af1, lymphoid profiles

**Review by sailepradh — COMMENTED** [review]
looks like a really big commit?

File: modules/local/ASCAT/main.nf [hunk 1]
```diff
@@ -1,76 +1,134 @@
...
-		tuple val(group), val(id), val(type), file("${group}.results.txt"), emit: ascat252_purplo
-
-	"""
-	Rscript /fs1/viktor/SomaticPanelPipeline_dsl2/bin/run_ascat.R 
-	Rscript /fs1/viktor/SomaticPanelPipeline_dsl2/workflows/bin/GenomicScars.R $params.scarfunctions $params.chrominfo_solid $params.chrominfo_general $params.fdata $group
-	cp HRD/${group}_HRD_Telli.txt .
-	mv ASCAT_summary_results_gamma0.9_penalty20.txt ${group}.results.txt
-	"""
-
+    publishDir "${params.outdir}/${params.subdir}/ASCAT2.5.2/", mode: 'copy', overwrite: true, pattern: '*.txt'
```
[the remark this answers is not in the document]
Comment (ViktorHy) [reply modules/local/ASCAT/main.nf:38]: ASCAT is deprecated. Would just remove this module

File: modules/local/GATK/main.nf [hunk 2]
```diff
@@ -1,371 +1,546 @@
...
+    input:
+        tuple val(group), file(seg), val(meta)
+
+    output:
+        tuple val(group), val(meta), file("${meta.id}_gatk_tumor.vcf"), emit: tumor_vcf
+        path "versions.yml",                                            emit: versions
+
+    script:
+        """
+        python3 /fs1/viktor/SomaticPanelPipeline_dsl2/bin/gatk_to_vcf.py -s $seg -o ${meta.id}_gatk_tumor.vcf -p ${meta.purity}
```
[the remark this answers is not in the document]
Comment (ViktorHy) [reply modules/local/GATK/main.nf:493]: Or at least just point to a interpreter in shebang. Should not be hardcoded like that

File: configs/nextflow.hopper.config
[the remark this answers is not in the document]
Comment (ramsainanduri) [reply configs/nextflow.hopper.config:1]: We can check for nf-core linting in the upcoming pull requests

File: modules/local/ASCAT/main.nf (hunk 1 above)
[the remark this answers is not in the document]
Comment (ramsainanduri) [reply modules/local/ASCAT/main.nf:38]: Yes, module level configs will be in the next iterations. ASCAT can be removed in the next iteration as well.

File: modules/local/GATK/main.nf (hunk 2 above)
[the remark this answers is not in the document]
Comment (ramsainanduri) [reply modules/local/GATK/main.nf:493]: Yes, will be another commit for these changes on the whole pipeline

**Pull request merged**